plugin.xml

```diff
@@ -2,9 +2,10 @@
 
 <plugin>
     <class>org.jivesoftware.openfire.plugin.SearchPlugin</class>
-    <name>Search</name>
-    <description>Provides support for Jabber Search (XEP-0055)</description>
-    <author>Ryan Graham</author>
+    <name>${project.name}</name>
+    <description>${project.description}</description>
+    <author>${project.developers[0].name}</author>
+    <licenseType>apache</licenseType>
```

Comment:
Not that I care much, but I _think_ that you can define the license of a project in the `pom.xml` file, too. Given that you're moving everything from `plugin.xml` to the pom, I thought I'd mention that.

Reply:
Ah, that's what you do in the next commit. Nevermind. :)